snap/hooks/configure

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -e
+
+$SNAP/bin/helper-go configure
```

Comment:
```suggestion
exec $SNAP/bin/helper-go configure
```
So that helper-go is executed as the same process rather than a child process of configure bash script. Snap wrapper scripts usually do that.

Same change for the install script.